mifospay/src/main/java/org/mifos/mobilewallet/mifospay/base/BaseActivity.java

```diff
@@ -145,6 +146,12 @@ public void hideBackButton() {
         }
     }
 
+    @Override
+    public void showWhiteBackButton() {
+        showHomeButton();
+        setToolbarIcon(R.drawable.ic_arrow_back_black_24dp);
```

Comment:
the function name is showWhiteBackButton but it sets a black icon?
Anyhow, the function name should be `showBackButton`

Reply:
@naman14 i am sorry i changed the color instead in ic_arrow_back_black_24dp.
Also also in mobile verification activity the back navigation color is white that's why i have used 2 function showWhiteBackButton for white back navigation icon and showBackButton for showing back color arrow navigation button.

Reply:
@naman14 should I change the name of the function to the corresponding color for back navigation as showBlackBackButton and for white showWhiteBack Button.